frontend/src/components/accountMgmt/ManageUsers.vue

```diff
@@ -1,6 +1,54 @@
 <template>
   <v-container fluid class="pa-12">
-    <h1 class="mb-6">Manage Users</h1>
+    <h1>Manage Users</h1>
+    <p class="mb-6">
+      <b>{{ organizationName }}</b> <br />
+      ID: {{ organizationAccountNumber }}
+    </p>
+    <v-row v-if="contactsLoading" no-gutters>
+      <v-col cols="12">
+        <v-card variant="outlined" class="soft-outline fill-height px-2">
+          <v-skeleton-loader type="paragraph" />
+        </v-card>
+      </v-col>
+    </v-row>
+    <v-row v-else>
+      <v-col>
+        <v-data-table
+          v-model:sort-by="sortBy"
+          :headers="headers"
+          :items="contacts"
+          item-key="contactId"
+          :items-per-page="10"
+          density="compact"
+          :mobile="null"
+          mobile-breakpoint="md"
+          class="soft-outline"
+        >
+          <template #[`item.edit-user`]="{ item }">
+            <v-row no-gutters class="my-2 align-center justify-end justify-md-start">
+              <AppButton :primary="false" size="small" @click="() => alert(item.clientId)">Edit</AppButton>
+            </v-row>
+          </template>
+          <template #[`item.access-type`]="{ item }">
+            <div v-if="isContactOnly(item)">Contact Only</div>
+            <div v-else class="d-flex justify-end justify-md-start flex-wrap">
+              Portal User
+              <v-chip v-if="isPrimaryContact(item)" class="ml-2" color="success" density="compact" variant="outlined">
+                Primary Contact
+              </v-chip>
+            </div>
+          </template>
+          <template #[`item.remove-user`]="{ item }">
+            <v-row no-gutters class="my-2 align-center justify-end">
+              <AppButton :primary="false" color="#F44336" size="small" @click="() => alert(item.clientId)">
```

Comment:
It looks like #d8292f is the most commonly used red in CCOF.

Reply:
Thank you